src/core/person.js

```diff
@@ -20,6 +20,10 @@ Person.prototype.variables = ['firstName', 'lastName', 'affiliation', 'publicati
 Person.prototype.variables.push.apply(Person.prototype.variables, Rankable.prototype.variables);
 Person.prototype.representation = PersonRepr;
 //gemiddelde aantal publicaties (per jaar) * (1 + publicaties dit jaar)
+/**
+ * Calculate the rank of the Person for which the function is called. This is calculated by: mean nr of published publications/year * (1 + nr of published publications this year)
+ * @return {Thenable<Number>} Thenable which holds the rank as a number
```

Comment:
Thenable? wa is er mis me promise? :p Btw ik heb overal al jsDoc bijgezet in req-pub11